ci/helm/preview/values.yaml

```diff
@@ -85,24 +85,25 @@ nuxeo:
         CREATE DATABASE nuxeo ENCODING 'UTF8' OWNER nuxeo;
     persistence:
       enabled: false
-    labels:
-      branch: "$BRANCH_NAME"
-      resource: deployment
-      team: napps
-      usage: preview
     resources:
       requests:
         cpu: 500m
         memory: 1Gi
       limits:
         cpu: 1
         memory: 2Gi
-    tolerations:
-    - key: team
-      operator: Equal
-      value: napps
-      effect: NoSchedule
-    nodeSelector:
-      team: napps
+    master:
+      labels:
+        branch: "$BRANCH_NAME"
```

Comment:
I don't know if we have a best practice on these conf, I mean should we put at the beginning of the file itself

Reply:
We can follow what did the platform team